src/webgpu/api/validation/texture/rg11b10ufloat_renderable.spec.ts

```diff
@@ -0,0 +1,53 @@
+export const description = `
+Tests for capabilities added by rg11b10ufloat-renderable flag.
+`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { ValidationTest } from '../validation_test.js';
+
+export const g = makeTestGroup(ValidationTest);
+
+g.test('create_texture_render_attachment')
+  .desc(
+    `
+Test that it is valid to create texture with rg11b10ufloat texture format and
+RENDER_ATTACHMENT usage is valid if rg11b10ufloat-renderable feature is enabled.
+`
+  )
+  .unimplemented();
+
+g.test('create_texture_multisampling')
+  .desc(
+    `
+Test that it is valid to create texture with rg11b10ufloat texture format and
+sampleCount > 1 if rg11b10ufloat-renderable feature is enabled.
+`
+  )
+  .unimplemented();
```

Comment:
Just for a little extra completeness, how about combining these two tests so they test 4 cases instead of just 3?

BTW when it's possible to write out the parameters with the plan, it makes it a bit easier to review test coverage completeness in the plan. For tests as simple as these, it's also totally fine to do both the plan and the implementation in one PR, though no harm in having a plan review!

```suggestion
g.test('create_texture')
  .desc(
    `
Test that it is valid to create rg11b10ufloat texture with RENDER_ATTACHMENT usage and/or
sampleCount > 1, iff rg11b10ufloat-renderable feature is enabled.
`
  )
  .params(u =>
    u
      .combine('enabled', [false, true])
      .combine('usage', [0, GPUConst.TextureUsage.RENDER_ATTACHMENT])
      .combine('sampleCount', [1, 4])
  )
  .unimplemented();
```

Reply:
> Just for a little extra completeness, how about combining these two tests so they test 4 cases instead of just 3?

That sounds good to me. Thanks for the suggestion.

>       .combine('enabled', [false, true])

If I'm right, disabled `rg11b10ufloat-renderable` feature test is not needed here beucause it's tested in https://github.com/gpuweb/cts/blob/main/src/webgpu/api/validation/createTexture.spec.ts

Reply:
Thanks!